### DISCUSSION TITLE
Is it posible to recreate a master node ?

### DISCUSSION THREAD

**Discussion by bobby0724** [okd-project/okd#1789] [Q&A]
**Describe the bug**

1 of my master nodes has been deleted
is there any way i can rebuild it ?

**Version**
4.12.0-0.okd-2023-04-01-051724

running in oVirt 4.4

**How reproducible**

**Log bundle**

**Comment by JaimeMagiera** [comment]
Here are instructions on replacing etcd members (control plan nodes) that are not running. 

https://docs.okd.io/4.13/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member

**Reply by felicianmv** [reply]
Hi Jaime,
That's exactly the procedure we followed, for both a master and a worker. For worker it completed after we updated the VM template, but master node is created and stays in Provisioned state, it never joins the cluster (and ovirt webadmin shows no ip for it, but we see an ip allocated in dhcp server logs).
